### DISCUSSION TITLE
Docker Compose Template?

### DISCUSSION THREAD

**Discussion by jodfie** [chuot/rdio-scanner#253] [Q&A] [answered]
Does anyone have a Docker Compose Template for the Rdio Docker container?

**Comment by chuot** [comment, marked as answer]
There's one in the [docs/examples/docker](https://github.com/chuot/rdio-scanner/blob/master/docs/examples/docker/docker-compose.yml) folder.

**Reply by jodfie** [reply]
Awesome! Sorry i missed that!

**Reply by chuot** [reply]
I must admit that the Docker documentation was missing since v6, but the container image was still built when releasing a new version.

So I made some improvements:

- `README.md` now has a Docker section
- `docs/docker` contains everything related to Docker, where `docs/docker/README.md` is the source document of what is on [hub.docker.com](https://hub.docker.com/repository/docker/chuot/rdio-scanner).

Hope this helps those who prefer the Docker flavor, like me which I use for my demo server.
